lib/chef/azure/commands/enable.rb

```diff
@@ -66,9 +66,9 @@ def enable_chef
     begin
       configure_chef_only_once
 
-      install_chef_service if @exit_code == 0
+      install_chef_service if @exit_code == 0 && windows?

```

Comment:
It looks like there is alot of ode duplication between these two methods and its not obvious that one is windows specific and the other linux. I would consolidate the two methods and have a platform conditional inside the service that calls the correct method to either impact the service system or chron.